mints/04.json

```diff
@@ -0,0 +1 @@
+{"name":"contract/valory/market_maker:0.1.0","description":"Fixed product market maker contract","code_uri":"ipfs://bafybeighn4o263yxjxc46n5rckfnonwtyyhvjmegmkrqraixvswuyhcsp4","image":"ipfs://bafybeievzsyvjc3oyuwrnlceionzkdbhydoripjeuvlngbkhahjix2c7vq","attributes":[{"trait_type":"version","value":"0.1.0"}]}
```

Comment:
Why "fixed"?

Reply:
This is how they are called:

https://github.com/protofire/omen-subgraph#fixedproductmarketmaker